wondrous-app/components/Common/TaskSubmission/styles.tsx

```diff
@@ -105,52 +107,76 @@ export const SubmissionButtonEdit = styled(ButtonBase)`
   }
 `;
 
-export const SubmissionButtonRequestChange = styled(ButtonBase)`
-  && {
-    border: 1px solid #474747;
-    width: max-content;
+const ButtonStyleSelected = css`
+  ${({ selected }) =>
+    selected &&
+    `
+  > button {
+    background: ${palette.background.default};
+    :hover {
+      background: ${palette.background.default};
+    }
+  }
+ `}
+`;
+
+const ButtonStyleBase = css`
```

Comment:
Please use button from `components/Button`

Reply:
We need a bit of customization to the buttons that use this. How can I make this code better?

Reply:
it's better to add a new definition to the theme in `components/Button/index.tsx` Here you can find `theme` constant and add your own colors:

```
const theme = {
  .....
  
  mynewcolor: {
    borderColor: palette.red200,
    background: palette.red200,
    borderRadius: 6,
    hover: {
      background: palette.grey920,
    },
  },
};
```

add new color as specifying it for the button, lie `<Buttin color="mynewcolor">`